site/reference/pkg/cat/README.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Cat"
+linkTitle: "cat"
+type: docs
+description: >
+  Print the resources in a file/directory
+---
+
+<!--mdtogo:Short
+    Print the resources in a file/directory
+-->
+
+Cat prints the resources in a file/directory as yaml to stdout.
```

Comment:
Style issue: the command name is `cat`, not `Cat`. Proper nouns should be in verbatim quotes.

But in this case, is kind of necessary to name the command:

`Prints resources in a file or directory...`

It's good to start a style guide to make sure we're consistent (Similar to CLI error/validation style guide).

Reply:
Updated.